compiler/shared/src/main/scala/Compiler.scala

```diff
@@ -183,6 +184,20 @@ object Compiler extends CompilerLike {
                             "notify"  -> jsFunction(Seq("str")),
                             "yesOrNo" -> jsFunction(Seq("str"), "return true;")))
 
+  private def fileReaderConfig: JsStatement =
+    genConfig("fileReader",
+      Map("read" -> jsFunction(
+                      Seq("filepath"),
+                      """var Paths = Java.type('java.nio.file.Paths');
+                        |var Files = Java.type('java.nio.file.Files');
+                        |var UTF8  = Java.type('java.nio.charset.StandardCharsets').UTF_8;
+                        |var lines = Files.readAllLines(Paths.get(filepath), UTF8);
```

Comment:
I think this is really cool, also I love the `java.nio.file` API and it's fun to see we're making use of it in Javascript.